frontend/angular.json

```diff
@@ -22,19 +22,20 @@
             "outputPath": "dist/frontend",
             "index": "src/index.html",
             "browser": "src/main.ts",
-            "polyfills": [
-              "zone.js"
-            ],
+            "polyfills": ["zone.js"],
             "tsConfig": "tsconfig.app.json",
             "inlineStyleLanguage": "scss",
             "assets": [
               {
                 "glob": "**/*",
                 "input": "public"
-              }
+              },
+              "./src/assets"
             ],
             "styles": [
-              "src/styles.scss"
+              "src/styles.scss",
+              "node_modules/primeng/resources/primeng.min.css",
+              "primeng/resources/themes/fluent-light/theme.css"
```

Comment:
If you're using PrimeNG, might be a better idea to use their SASS theme and bundle it in with the custom `src/styles`. 
Move the styles file into a subfolder under src. E.g. `src/styles/styles.scss`

Reply:
i'm unfamiliar with this, skipping it for now if it's ok.